Algorithms/Algorithms.Tests/KnuthMorrisPrattTests/KnuthMorrisPrattTests.cs

```diff
@@ -0,0 +1,25 @@
+﻿using Xunit;
+
+namespace Algorithms.Tests
+{
+    public class KnuthMorrisPrattTests
+    {
+        [Fact]
+        public void FunctionalityCheck()
```

Comment:
It's better to provide single [Theory] with multiple [Inline] tests.

Reply:
@Rizzen Done